programs/psinode/main.cpp

```diff
@@ -1948,6 +1948,7 @@ void run(const std::string&              db_path,
                          result = std::make_shared<PKCS11Prover>(pos->second.session, key.service);
                       }
                       pos->second.sessionProver->add(result);
+                      node.on_key_update();
```

Comment:
This not needed in the `key.service == AccountNumber{"verify-sig"}` path?

Reply:
No. That's the path that stores the key in the config file, which is handled at at L2007.